lib/iex/lib/iex/info.ex

```diff
@@ -430,6 +430,30 @@ defimpl IEx.Info, for: [Date, Time, DateTime, NaiveDateTime] do
   end
 end
 
+defimpl IEx.Info, for: Range do
+  def info(value) do
+    description = """
+    This is a struct representing a range of numbers. It is commonly
+    represented using the `first..last//step` syntax. The step is not
+    required and defaults to 1.
+    """
+
+    [
+      {"Data type", inspect(@for)},
+      {"Description", description},
+      {"Raw representation", raw_inspect(value)},
+      {"Reference modules", inspect(@for), "Map"}
+    ]
+  end
+
+  defp raw_inspect(value) do
+    value
+    |> Inspect.Any.inspect(%Inspect.Opts{})
+    |> Inspect.Algebra.format(:infinity)
+    |> IO.iodata_to_binary()
+  end
```

Comment:
This function is already implemented in this file, but since there is no module defined, I reimplemented it.